### PR TITLE
New package: cuda-7.0.28

### PR DESCRIPTION
**Pull request by tonkazoid**
sorry for the fuss

### CODE REVIEW

File: srcpkgs/cuda/template [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Template file for 'cuda'
+pkgname=cuda
+version=7.0.28
+revision=1
+maintainer="Georg Schabel <gescha@posteo.de>"
+homepage="http://www.nvidia.com/object/cuda_home.html"
+short_desc="NVIDIA's GPU programming toolkit"
+license="Propietary NVIDIA license"
```
Comment (Gottox) [inline srcpkgs/cuda/template]: typo: "proprietary"

**Comment by Gottox** [comment]
Does this licence allow redistribution?

**Comment by tonkazoid** [comment]
http://docs.nvidia.com/cuda/eula/index.html#redistribution-rights
http://docs.nvidia.com/cuda/eula/index.html#linux-freebsd-exception

Oh and the "propietary". Blindly Copied from here https://github.com/voidlinux/void-packages/blob/ff31d5a1a200f252a6a14db42eef2269265199fb/srcpkgs/nvidia/template#L9

**Comment by xtraeme** [comment]
This can't be applied again, no idea why.

**Comment by tonkazoid** [comment]
Strange.. 
Another thing i've noticed: I'm currently using it without a problem in blender. But after the first installation i need to start blender as root once for the GPU/CPU selection to show up for all users. I don't know if i can fix this with the template.

**Comment by tonkazoid** [comment]
most people fix the "run as root once"-problem with nvidia-modprobe. added it to the nvidia package and removed the libavcodec53 from shlibs, this package doesn't exist anymore.

**Pull request closed without merging**